hardware/neopixel/neopixel.js

```diff
@@ -102,7 +107,7 @@ module.exports = function(RED) {
             }
         }
 
-        node.child = spawn(piCommand, [node.pixels, node.wipe, node.mode]);
+        node.child = spawn(piCommand, [node.pixels, node.wipe, node.mode, node.gamma,node.brightness]);
```

Comment:
if gamma can be omitted  (not defaulted) - may a s well make it the last param instead... easier to skip

Reply:
AFAICT gamma cannot be omitted but I'll swap them around anyway

Reply:
order swapped